package.json

```diff
@@ -31,6 +31,7 @@
     "after": "0.8.1",
     "istanbul": "0.3.9",
     "mocha": "2.2.5",
+    "replacestream": "~4.0.0",
```

Comment:
Yeah, I'm looking at using an older version of `replacestream` which I think will fix this (a version that has no dependencies specified by caret version).